tests/test_get_sub_criteria.py

```diff
@@ -1,11 +1,36 @@
 from unittest.mock import Mock
 
+import pytest
 from api.routes.subcriterias.get_sub_criteria import (
     sort_add_another_component_contents,
 )
 
 
-def test_sort_add_another_component_contents():
+@pytest.mark.parametrize(
+    "answer_from_form_runner, expected_description, expected_amount",
+    [
+        (
+            ["An Answer : £1"],
+            ["An Answer "],
+            ["£1"],
+        ),
+        (
+            ["An Answer : with colon : £1"],
+            ["An Answer : with colon "],
+            ["£1"],
+        ),
+        (
+            ["An Answer : £1", "Another Answer : £2"],
```

Comment:
Can we add a test with multiple `:` characters in the description part? We should be able to handle any number and only the last one should be the amount.

Reply:
This is being done just above on [L18](https://github.com/communitiesuk/funding-service-design-assessment-store/blob/023feaf7c3ff9f76bf157ec40f157dd5827b35aa/tests/test_get_sub_criteria.py#L18) - or do you mean even more colons?